src/puppetlabs/pcp/client.clj

```diff
@@ -196,7 +196,8 @@
                         :on-error (fn [error]
                                     (log/error error (i18n/trs "WebSocket error")))
                         :on-close (fn [code message]
-                                    (log/debug (i18n/trs "WebSocket closed {0} {1}" code message))
+                                    ;; Format error code as a simple string (rather than localized).
+                                    (log/debug (i18n/trs "WebSocket closed {0} {1}" (str code) message))
```

Comment:
I wonder what is the effect of this change?
The arguments must be coerced to strings sooner or later anyway.